semantic-ui-react-redux/src/main/scala/demo/Demo.scala

```diff
@@ -0,0 +1,55 @@
+package demo
+
+import demo.ReduxFacade.Connected
+import japgolly.scalajs.react.raw.React
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.{CallbackTo, ScalaFnComponent}
+import typings.semanticUiReact.anon.MenuItem
+import typings.semanticUiReact.cardCardMod.CardProps
+import typings.semanticUiReact.headerHeaderMod.HeaderProps
+import typings.semanticUiReact.progressProgressMod.ProgressProps
+import typings.semanticUiReact.searchSearchMod.SearchProps
+import typings.semanticUiReact.semanticUiReactStrings.large
+import typings.semanticUiReact.tabTabMod.TabProps
+import typings.semanticUiReact.{semanticUiReactStrings, components => Sui}
+
+import scala.scalajs.js
+
+object Demo {
+
+  val CardDemo: React.Element =
+    Sui.Card.withProps(CardProps().setColor(semanticUiReactStrings.orange))(
+      Sui.CardHeader("CardHeader"),
+      Sui.CardMeta("CardMeta"),
+      Sui.Divider(),
+      Sui.Search.withProps(SearchProps().setMinCharacters(1))
+    ).rawElement
+
+  val ProgressDemo: React.Element =
+    Sui.Card(
+      Sui.Progress.withProps(ProgressProps().setPercent(70).setWarning(true)),
+      Sui.Progress.withProps(ProgressProps().setPercent(100).setWarning(false)),
+    ).rawElement
+
+
+  class Props(val title: String) extends js.Object
+
+
+  val component = ScalaFnComponent[Connected[GithubSearch.State, GithubSearch.SearchAction] with Props] { props =>
+    <.div(
+      Sui.Header.withProps(HeaderProps().setSize(large))(props.title),
+      Sui.Tab.withProps(TabProps().setPanes(js.Array(
```

Comment:
there is new syntax for this now, you should be able to say `Sui.Tab.panesVarargs(MenuItem()...)`